api.js

```diff
@@ -0,0 +1,17 @@
+async function login(email, password) {
+  const response = await fetch('/api/login', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ email, password }),
+  });
+
+  if (response.ok) {
+    return true;
+  } else {
+    return false;
```

Comment:
The login function should probably return more than just a boolean. If the login fails, it would be useful to know why so that the user can be informed. Consider returning the entire response object, or at least the status code and error message if there is one.